app/controllers/vaccines_controller.rb

```diff
@@ -1,8 +1,14 @@
 class VaccinesController < ApplicationController
   include TodaysBatchConcern
 
+  layout "two_thirds"
```

Comment:
This doesn't look quite right , but it's broken in the same way other pages are broken ... I think the prototype uses a three-quarters layout? 🥾 🥫

Reply:
Kick the can yeah, we don't have the 3-quarters layout yet, asked on Slack about it